app/src/components/createLearningJourney/index.jsx

```diff
@@ -11,16 +11,19 @@ import { createLJCourseMapping } from "src/api/learningJourneyCourse";
 import JobSkills from "./JobSkills";
 import CoursesList from "./CoursesList";
 import CourseModal from "./CourseModal";
-import SubmitButton from "./SubmitButton"
+import SubmitButton from "./SubmitButton";
+import DeleteSkillModal from "./DeleteSkillModal";
 
 export default function index() {
   const [isModalOpen, setIsModalOpen] = useState(false);
   const [coursesAndSkillsMapping, setCoursesAndSkillsMapping] = useState([]);
   const [currentSelectedSkill, setCurrentSelectedSkill] = useState("");
   const [errorMessage, setErrorMessage] = useState("");
+  const [isDeleteSkillModalOpen, setDeleteSkillModalOpen] = useState(false);
 
   const history = useHistory();
-  const { selectedJobRole, clearSelectedCourseDetails, selectedCourseDetails } = useLJCreationContext();
+  const { selectedJobRole, clearSelectedCourseDetails, selectedCourseDetails } =
```

Comment:
Note this part will be breaking. Your PR can gofirst, I'll handle merge conflict after because I changed some naming.

let me know once u merged.